stdpopsim/species.py

```diff
@@ -113,6 +115,77 @@ class Species(object):
     demographic_models = attr.ib(factory=list, kw_only=True)
     genetic_maps = attr.ib(factory=list, kw_only=True)
 
+    def get_chunk(self, length, chromosome=None, position=None, genetic_map=None):
```

Comment:
I'm not sure we need this: why not add ``start`` and ``end`` parameters to the ``get_contig`` method?